cypress/integration/userCanCreateRecipe.spec.js

```diff
@@ -0,0 +1,36 @@
+describe('Creates a recipe', () => {
+
+  it('Successfully', () => {
+    cy.route({
+      method: 'POST',
+      url: 'http://localhost:3000/v1/recipes',
+      response: '{ "message": "The recipe was successfully created." }',
+      status: 201
+    })
+  
```

Comment:
You have to `visit` the app

Reply:
Oh, you visit it in a before block in `index.js` Well, the problem is that the stub won't kick in (I think)

Reply:
Stub kicks in for me and on Sverrir's computer.